ci/infra/openstack/security-groups-load-balancer.tf

```diff
@@ -0,0 +1,35 @@
+resource "openstack_networking_secgroup_v2" "load_balancer" {
+  name        = "${var.stack_name}-caasp_common_secgroup"
+  description = "Common security group for CaaSP load balancer"
+}
+
+resource "openstack_networking_secgroup_rule_v2" "lb_api_server" {
+  direction         = "ingress"
+  ethertype         = "IPv4"
+  protocol          = "tcp"
+  port_range_min    = 6443
+  port_range_max    = 6443
+  remote_ip_prefix  = "0.0.0.0/0"
+  security_group_id = "${openstack_networking_secgroup_v2.load_balancer.id}"
+}
+
+# Range of ports used by kubernetes when allocating services of type `NodePort`
```

Comment:
NodePort are open directly the nodes so there is no need to add them on the LB so having this in the sg `common` is enough IMO.

By default we need TCP `32000` for dex and TCP `32001` for Gangway.

Reply:
You're right. I fixed that